.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: node_js
 node_js:
-  - '8.10'
-  - '10'
+  - '10.16.3' # lambda version as of 2019-11-21
```

Comment:
no joke, AWS release node 12 support this week. We should add support for that soon, but not today